### DISCUSSION TITLE
[BUG | Galaxy Brain] When there're marked answer related to your own question, the achievement will not be displayed as achieved.

### DISCUSSION THREAD

**Discussion by qhy040404** [community/community#18508] [Other Feature Feedback, Questions, & Ideas]
Galaxy Brain's trigger has bugs. 
When there are marked answers that are marked by myself, the achievement will not be displayed as completed. After unmarking my answers, I can get the achievement after my answer was marked by other people. 
Of course, this avoids some invalid data. However, the possibility of sharing the solution after solving the problem by myself cannot be ruled out. 
> anuraghazra/github-readme-stats#1765
> 
> I unmarked the answer temporarily to test.

**Comment by TSH3PH4NG** [comment]
hmm 🙂 i want one

**Reply by Caypoh23** [reply]
agree
